server/build-plugins.js

```diff
@@ -33,6 +35,47 @@ const getPackageName = name => {
   }
 };
 
+async function installRemotePackages(installPackages) {
+  // check if connected to internet
+  // if not, skip npm install
+  const EXTERNAL_URI = process.env.EXTERNAL_URI || 'npmjs.com';
+  await require('dns').lookup(EXTERNAL_URI, async err => {
+    if (err && err.code === 'ENOTFOUND')
+      logger.error(`Can't reach npm servers. Skipping npm install`);
+    else {
+      logger.info('Installing plugin packages...');
+      await execSync(
+        `npm install --no-save ${installPackages.join(' ')} --production`,
+        {
+          stdio: [0, 1, 2],
+          cwd: resolve(__dirname, '..')
+        }
+      );
+      logger.info('Done installing plugins');
+    }
+  });
+}
+
+function symlinkLocal(packageName) {
```

Comment:
As noted in the comment where this is invoked, we symlink the local plugins to the local node_modules so that webpack can watch for changes to plugins rather than have nodemon watch an external directory and trigger a restart of the whole webpack process which would take longer.

Reply:
Just reading from the top, I didn't see the comment where it was invoked until I got to that part of the page haha